vercel-api/api/publish.ts

```
@@ -183,28 +159,26 @@ export default async function handler(
       });
 
       res.status(201).json(successResponse);
+      
+      // Record successful request and WordPress API call
+      const processingTime = Date.now() - startTime;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'processingTime'. Consider to delete it or rename it.

'processingTime' is defined here:


(lint/suspicious/noRedeclare)